src/main/java/com/dft/veracross/model/students/studentWrapper/StudentData.java

```diff
@@ -0,0 +1,39 @@
+package com.dft.veracross.model.students.studentWrapper;
+
+import com.dft.veracross.model.students.StudentsInfo;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class StudentData {
+
+    public Integer id;
+    public Integer namePrefix;
+    public String firstName;
+    public Object middleName;
```

Comment:
Provide proper data type please. for all Object

Reply:
> Provide proper data type please. for all Object

Provided proper datatype in everyclass.